tests/js/spec/views/performance/transactionSummary/teamKeyTransactionButton.spec.jsx

```diff
@@ -297,8 +297,6 @@ describe('TeamKeyTransaction', function () {
       await tick();
       wrapper.update();
 
-      clickTeamKeyTransactionDropdown(wrapper);
```

Comment:
Was the checkbox click before closing the dropdown and you had to re-open?

Reply:
Yes